src/test/resources/freemarker/manual/CustomFormatsExample-alias1.ftlh

```diff
@@ -16,6 +16,8 @@
   specific language governing permissions and limitations
   under the License.
 -->
+<#setting locale="en_us">
```

Comment:
Indeed, but then the `locale` and `time_zone` should be set in the `ExamplesTest` class, rather that in each template. Especially because otherwise the examples can't be copy-paster from/to the Manual.